src/main/java/frc/robot/utils/logging/SequentialLoggingCommand.java

```diff
@@ -29,6 +47,11 @@ public String toString() {
         return getFullyQualifiedName();
     }
 
+    // For testing
+    public List<LoggingCommand> getLoggingCommands() {
```

Comment:
this says "For testing" do we still need it?

Reply:
Yes - I intend to add the tests I wrote to the common repo

Reply:
Here are the tests for the CommandUtil: https://github.com/FRC4048/java-common/pull/11